textdb/textdb-textql/src/main/java/edu/uci/ics/textdb/textql/statements/predicates/SelectPredicate.java

```diff
@@ -5,8 +5,8 @@
 /**
  * Object representation of a "SELECT (...)" predicate inside a { @code SelectExtractStatement }.
  * Subclasses have specific fields related to its projection functionalities.
- * SelectPredicate --+ SelectAllPredicate
- *                   + SelectFieldsPredicate
+ * SelectPredicate --+ SelectAllFieldsPredicate
```

Comment:
I was originally inclined to use Fields in this case because the SelectPredicate is heavily associated with the ProjectionOperator, and the ProjectionPredicate refeers to  'projectionFields' as a list of attribute names. If we decide to change from 'field' to 'attribute', I would suggest to make the same changes to the `ProjectionOperator` to make it consistent.
The projection does not take into account the 'fieldType' of an attribute, only the 'fieldName' is relevant for it.
I just did a quick search: most databases documentation refers to 'column' instead of 'attribute' or 'field', however, as @zuozhiw said, a field is often associated to a 'cell', the use of 'field' would be more adequate if we think of a cell as part of a column.
I don't have a strong opinion about it though. It would be great to have more oppinions and keep deliberating. Any other thoughts are welcome!